### PR TITLE
[Knowledge] `sizecalc.com` broken link deleted

### PR DESCRIPTION
**Pull request by vv-monsalve**
The [sizecalc.com site](http://sizecalc.com/) has stopped working. This PR deletes the broken link from the Knowledge content to solve the [issue](https://github.com/google/fonts/actions/runs/9785739276/job/27019312788#step:5:74) reported in the CI of recent PRs.

### CODE REVIEW

**Comment by vv-monsalve** [comment]
I've asked Nick about the website, and he will check to bring it back. So, let's hold on to this PR before merging it.

**Comment by vv-monsalve** [comment]
Link updated using the web archive

File: cc-by-sa/knowledge/modules/using_type/lessons/the_complications_of_typographic_size/content.md [hunk 1]
```diff
@@ -84,4 +84,4 @@ Despite some of the limitations and complications mentioned above, there’s sti
 - Be more observant about how different typefaces may require changes in size specifications due to differences in their design or production.
 - Perhaps more important than anything, test the sizing logic on a variety of screens and devices to ensure it works reasonably well in as many environments as possible.
 
-The more knowledgeable and thoughtful designers are to the intricacies of typographics size, the better the experience can be for readers.
\ No newline at end of file
+The more knowledgeable and thoughtful designers are to the intricacies of typographics size, the better the experience can be for readers.
```
Comment (davelab6) [inline cc-by-sa/knowledge/modules/using_type/lessons/the_complications_of_typographic_size/content.md:87]: I believe this shows an empty last line was removed, which is recommended for unix text files, please add back

Comment (vv-monsalve) [reply cc-by-sa/knowledge/modules/using_type/lessons/the_complications_of_typographic_size/content.md:87]: Actually the new version has a new empty line (#88)

**Review by davelab6 — APPROVED** [review]

File: cc-by-sa/knowledge/modules/using_type/lessons/the_complications_of_typographic_size/content.md (hunk 1 above)
Comment (davelab6) [reply cc-by-sa/knowledge/modules/using_type/lessons/the_complications_of_typographic_size/content.md:87]: Perfect then

**Pull request merged**